helix-core/src/main/java/org/apache/helix/monitoring/mbeans/PerInstanceResourceMonitor.java

```diff
@@ -113,6 +118,10 @@ public String getResourceName() {
     return _resourceName;
   }
 
+  public String getBeanName() {
```

Comment:
Is this method used anywhere? And the return string seems not to be a valid MBean name, which should look like, "%s,%s=%s,%s=%s"